app/javascript/src/components/Dashboard/Notes/Pane/Form.jsx

```diff
@@ -40,15 +41,65 @@ export default function NoteForm({ onClose, refetch, note, isEdit }) {
               label="Title"
               name="title"
               className="w-full flex-grow-0"
+              placeholder="Enter a title"
               required
             />
             <Textarea
               label="Description"
               name="description"
               className="w-full flex-grow-0"
-              rows={8}
+              rows={2}
+              placeholder="Enter note description"
               required
```

Comment:
Implicitly true values should be placed at top of the attribute list.

Please read [this section](https://www.bigbinary.com/react-best-practices/miscellaneous-tips#:~:text=Also%20note%20that%20these%20attributes%20ought%20to%20be%20the%20first%20among%20the%20list%20of%20attributes%20being%20passed%20down%20as%20props%20because%20it%20gives%20a%20better%20visual%20hierarchy%20and%20also%20these%20are%20the%20props%20that%20don%27t%20change%20often.).